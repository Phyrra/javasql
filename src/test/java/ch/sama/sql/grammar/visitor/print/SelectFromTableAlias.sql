select alias.*
from t as alias