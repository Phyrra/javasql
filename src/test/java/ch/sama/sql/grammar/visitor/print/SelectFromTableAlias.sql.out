sql
	statement
		select
			value
				all
					alias
		from
			source
				aliased
					table
						t
					alias
